x-pack/plugins/transform/public/app/services/es_index_service.ts

```diff
@@ -5,8 +5,8 @@
  * 2.0.
  */
 
-import { HttpSetup, SavedObjectsClientContract } from '@kbn/core/public';
-import { DataView } from '@kbn/data-views-plugin/public';
+import { HttpSetup } from '@kbn/core/public';
+import { DataViewsContract } from '@kbn/data-views-plugin/public';
```

Comment:
```suggestion
import type { DataViewsContract } from '@kbn/data-views-plugin/public';
```

Reply:
Done in https://github.com/elastic/kibana/pull/131819/commits/36d639885f4609acb2982b083b9639e7a47b7956